modules/node_modules/@ncigdc/dux/relayLoading.js

```diff
@@ -0,0 +1,20 @@
+const TOGGLE_LOADING = 'TOGGLE_LOADING'
+
+const toggleLoading = isLoading => ({
+  type: TOGGLE_LOADING,
+  payload: isLoading,
+})
+
+const relayLoading = (state = null, action) => {
+  switch (action.type) {
+    case 'TOGGLE_LOADING':
+      return action.payload
+    default:
+      return state
+  }
+}
+
+/*----------------------------------------------------------------------------*/
+
+export default relayLoading
```

Comment:
It *was* for initial page loading, I just copied over the folder from legacy branch, so will likely remove some of these as relay takes over the need for it